changelog/2394.trivial.rst

```diff
@@ -0,0 +1 @@
+Replaced `black` with `ruff` in pre-commit config.
```

Comment:
```suggestion
Replaced |black| with |ruff| in pre-commit config.
```
We haven't configured instersphinx for `black` and `ruff` to our docs, but instead have been using reStructuredText substitutions via [`sphinxcontrib-globalsubs`](https://github.com/missinglinkelectronics/sphinxcontrib-globalsubs), with the reStructuredText substitutions set up in [`docs/_global_substitutions.py`](https://github.com/PlasmaPy/PlasmaPy/blob/main/docs/_global_substitutions.py).